geometric_kernels/spaces/homogeneous_spaces.py

```diff
@@ -0,0 +1,155 @@
+import lab as B
+import json
+from pathlib import Path
+import more_itertools
+import sympy
+from sympy.matrices.determinant import _det as sp_det
+import numpy as np
+from geometric_kernels.lab_extras import dtype_double, from_numpy
+from geometric_kernels.spaces.base import DiscreteSpectrumSpace
+from geometric_kernels.spaces.eigenfunctions import EigenfunctionWithAdditionTheorem, Eigenfunctions
+from geometric_kernels.spaces.lie_groups import LieGroup
+import abc
+
+
+class HomogeneousSpaceAddtitionTheorem(EigenfunctionWithAdditionTheorem):
```

Comment:
`HomogeneousSpaceAddtitionTheorem` > `HomogeneousSpaceAdditionTheorem` (note the typo)